src/main/views/crown-daily-list.njk

```diff
@@ -0,0 +1,190 @@
+                    {% endif %}
+                </span>
+                </h2>
+              </div>
+              <div id="accordion-default-content-{{ sessionCount }}" class="govuk-accordion__section-content" aria-labelledby="accordion-default-heading-{{ sessionCount }}">
+                <div class="parent-box overflow-table">
+                  <table class="govuk-table">
+                    <thead class="govuk-table__head">
+                    <tr class="govuk-table__row">
+                      <th scope="col" class="govuk-table__header no-wrap">{{ sittingAt }}</th>
+                      <th scope="col" class="govuk-table__header no-wrap">{{ caseReference }}</th>
+                      <th scope="col" class="govuk-table__header no-wrap">{{ defendantName }}</th>
+                      <th scope="col" class="govuk-table__header no-wrap">{{ hearingType }}</th>
+                      <th scope="col" class="govuk-table__header no-wrap">{{ duration }}</th>
+                      <th scope="col" class="govuk-table__header no-wrap">{{ prosecutingAuthority }}</th>
+                    </tr>
+                    </thead>
+
+                    <tbody class="govuk-table__body">
+                    {% for sitting in session['sittings'] %}
+                      {% set hearingPlatform = '' %}
+                      {% if sitting['channel'].length === 0 %}
+                        {% set hearingPlatform = sessionChannel %}
+                      {% else %}
+                        {% for channel in sitting['channel'] %}
+                          {% set hearingPlatform = hearingPlatform + ', ' + channel %}
+                        {% endfor %}
+                        {% set hearingPlatform = hearingPlatform.slice(1) %}
+                      {% endif %}
+                      {% for hearing in sitting['hearing'] %}
+                        {% for case in hearing['case'] %}
+                          {% if courtRoom['totalHearing'] === hearingCount %}
+                            {% set noBorder = 'no-border-bottom' %}
+                          {% endif %}
+                          <tr class="govuk-table__row">
+                            <td class="govuk-table__cell no-wrap {{ noBorder }}">{{ sitting['time'] }}</td>
+                            <td class="govuk-table__cell no-wrap {{ noBorder }}">{{ case['caseNumber'] }}</td>
+                            <td class="govuk-table__cell no-wrap {{ noBorder }}">{{ hearing['defendant'] }}</td>
+                            <td class="govuk-table__cell no-wrap {{ noBorder }}">{{ hearing['hearingType'] }}</td>
+                            <td class="govuk-table__cell no-wrap {{ noBorder }}">{{ sitting['formattedDuration'] }} {{ case['caseSequenceIndicator'] }}</td>
+                            <td class="govuk-table__cell no-wrap {{ noBorder }}">{{ hearing['prosecutingAuthority'] }}</td>
+                          </tr>
+                          {% if case['linkedCases'] | length %}
```

Comment:
Just testing this out now. It looks like the horizontal line seperates the linked cases too, where as the example in the ticket only has the horizontal line after the linked cases, so it groups them together. Can this be updated?

Also Linked Cases / Notes are bold here but not in example

Reply:
Good spot :) It is fixed now.